results.go

```diff
@@ -122,6 +124,34 @@ func (v *vocdoniHandler) finalizeElectionResults(election *api.Election, electio
 			return
 		}
 		log.Infow("final results image built ondemand", "electionID", election.ElectionID.String())
+		hubResults := &communityhub.HubResults{
```

Comment:
You should check if the election is part of a community that exist on the smart contract.